src/data/list/pairwise.lean

```diff
@@ -5,6 +5,14 @@ Authors: Mario Carneiro
 -/
 import data.list.sublists
 
+/-!
+# Pairwise relations on a list
+
+This file provides basic results about `list.pairwise`. `pairwise r [a 0, ..., a n]` means
+`∀ i j, i < j → r (a i) (a j)`. For example, `pairwise (≠) l` means that all elements of `l` are
+distinct, and `pairwise (<) l` means that `l` is strictly increasing.
+-/
+
```

Comment:
I think it would be nice to add some tags to this file, for instance referencing `pairwise`, `sorted` and `nodup`, since some lemmas about those are very closely related to things here.

Reply:
When exactly do we want to add tags? Because here lemma names already contain `pairwise`. For `sorted` and `nodup`, I agree.

Reply:
I tend to still add related keywords even if they already appear in the file, so that e.g. we can find them back after the definition gets renamed. I agree that this is not really necessary and we don't need to add `pairwise` here.